kedro-datasets/CONTRIBUTING.md

```diff
@@ -66,14 +102,19 @@ We use a branching model that helps us keep track of branches in a logical, cons
 | `fix`            | Non-breaking change which fixes an issue                                    |
 | `tests`          | Changes to project unit (`tests/`) and / or integration (`features/`) tests |
 
-## Plugin contribution process
+## Dataset contribution process
 
  1. Fork the project
  2. Develop your contribution in a new branch.
- 3. Make sure all your commits are signed off by using `-s` flag with `git commit`.
- 4. Open a PR against the `main` branch and sure that the PR title follows the [Conventional Commits specs](https://www.conventionalcommits.org/en/v1.0.0/) with the scope `(datasets)`.
- 5. Make sure the CI builds are green (have a look at the section [Running checks locally](#running-checks-locally) below)
- 6. Update the PR according to the reviewer's comments
+ 3. Add your dataset to the appropriate location:
+    - `kedro-datasets/experimental` for experimental datasets.
```

Comment:
Reading this made me realize that I'm not 100 % sure whether we decided to _ship_ these datasets as part of the `kedro-datasets` PyPI package or not. If we said "yes", then maybe we need to tweak either our packaging or the location of this directory. If we said "no", then it would be good to clarify it in this guide and https://github.com/kedro-org/kedro-plugins/blob/main/kedro-datasets/experimental/README.md.

Reply:
Good point! I think we should ship them to make it as easy as possible to use them e.g. just import `from kedro_datasets import experimental`.

Reply:
The results of the poll was a unanimous vote from 11 TSC members to include experimental contributions in the `kedro-datasets` package. I'll create a follow up task to update our packaging.

Reply:
> Good point! I think we should ship them to make it as easy as possible to use them e.g. just import `from kedro_datasets import experimental`.

I think it's fine to even have them be importable the same way/top-level `kedro_datasets` namespace. As in, your catalog entry doesn't have to be `experimental.sap.HANADataset`. Although I'm not 100% sure on this, maybe there is value to being explicit to the reader that they're using something coming from `experimental`.